src/ui/App/App.tsx

```diff
@@ -172,6 +174,17 @@ function PageLayoutViews() {
     <Routes>
       <Route path="/mint-dna/*" element={<MintDnaFlow />} />
       <Route path="/upgrade-dna/*" element={<UpgradeDnaFlow />} />
+
+      <Route
+        path="/backup/*"
+        element={
+          <RequireAuth>
+            <PageLayout>
+              <Backup />
+            </PageLayout>
+          </RequireAuth>
+        }
```

Comment:
I wanted to move the usage of <PageLayout> inside the <Backup> component, but in the <Onboarding> component, all these elements are explicitly wrapped (as if it is possible for these components to be used outside of <PageLayout> as well):
```
<Route                        
  path="/onboarding"          
  element={                   
    <PageLayout>              
      <Welcome />             
    </PageLayout>             
  }                           
/>                            
<Route                        
  path="/onboarding/create/*" 
  element={                   
    <PageLayout>              
      <Create />              
    </PageLayout>             
  }                           
/>                            
```
So I decided to leave it as is

Reply:
+
I hope it is a bit more flexible to use them this way tbh